data/assets/scene/digitaluniverse/multiwavelength_sky.asset

```diff
@@ -0,0 +1,64 @@
+-- local speck = asset.resource({
+--   Name = "Multiwavelength Milky Way Data Files",
+--   Type = "HttpSynchronization",
+--   Identifier = "digitaluniverse_multiwavelength_milkyway_data",
+--   Version = 1
+-- })
```

Comment:
If we don't need the resource, we should delete the lines entirely

Reply:
This is likely left in by mistake. I had aspirations to add the rest of the all-sky images, but realized only rectangular images work. So, I put in a map but then had to remove it, guess I left this there...